docs/topics/auth/customizing.txt

```diff
@@ -259,12 +259,48 @@ Handling object permissions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Django's permission framework has a foundation for object permissions, though
-there is no implementation for it in the core. That means that checking for
-object permissions will always return ``False`` or an empty list (depending on
-the check performed). An authentication backend will receive the keyword
-parameters ``obj`` and ``user_obj`` for each object related authorization
+there is no implementation for it in the core. An authentication backend will
+receive the keyword parameter ``obj`` for each object related authorization
 method and can return the object level permission as appropriate.
 
+The default :class:`~django.contrib.auth.backends.ModelBackend` only handles
+global permissions. That means that checking for object permissions will always
+return ``False`` or an empty set (depending on the check performed).
+Conversely, it is expected that any backend that handles object permissions
+returns ``False`` if no object is passed.
+
+Putting it all together, here is an example authentication backend handling
+object permissions::
+
+    class BlogPost(models.Model):
+        owner = models.ForeignKey(User, on_delete=models.CASCADE)
+        ...
+
+    class BlogPostBackend:
+        def authenticate(self, request, **kwargs):
+            return None
```

Comment:
This could be part of building the example — we have `ModelBackend` in play, which handles authentication. 

But (I'm getting there slowly... 🙂) we should be showing the `AUTHENTICATION_BACKENDS` setting (at each stage) as part of showing _how to use the system_ not **just** showing the backend implementation in a vacuum. (Especially given the new backend and permission mixin stuff — we need to show how they combine.)